sdcm/nemesis.py

```diff
@@ -280,8 +280,11 @@ def _choose_file_for_destroy(self, ks_cfs):
                 continue
             file_path_splitted = file.split('/')
             # The file name like: /var/lib/scylla/data/scylla_bench/test-f60e4f30c98f11e98d46000000000002/mc-220-big-Data.db
-            # For corruption we need to remove all files that their names are started from "mc-220-"
-            file_name = '-'.join(file_path_splitted[-1].split('-')[:2]) + '-*'
+            # For corruption we need to remove all files that their names are started from "mc-220-" (MC format)
```

Comment:
need improvement here 
this is unreadable: file_path_splitted[-1][:position+2]
use specific os functions for this
```
In [6]: os.path.basename("/var/lib/scylla/data/scylla_bench/test-f60e4f30c98f11e98d46000000000002/mc-220-big-Data.db")
Out[6]: 'mc-220-big-Data.db'
```
use normal regex to find the sstable generation number:

```
In [19]: re.search("mc-(\d+)-big.*", sstable_file).group(1)
Out[19]: '220'

```

Reply:
```
In [7]: re.search("(.*-\d+)-", "mc-220-big-Data.db").group(1)
Out[7]: 'mc-220'

```

Reply:
Fixed